src/emulator/init.cpp

```diff
@@ -118,11 +105,11 @@ bool init(HostState &state, Config cfg) {
     if (state.cfg.wait_for_debugger) {
         state.kernel.wait_for_debugger = state.cfg.wait_for_debugger.value();
     }
-    state.base_path = base_path.get();
+    state.base_path = root_paths.get_base_path_string();
 
     // If configuration already provides preference path
-    if (state.cfg.pref_path.empty()) {
-        state.pref_path = pref_path.get();
+    if (!state.cfg.pref_path.empty()) {
```

Comment:
wrong here, remove "!", like that your remplace value inside pref_path if user use external folder.

Reply:
Done. That was an spot-check failure on my part. *doh*